views/devices_list.pug

```diff
@@ -45,8 +39,7 @@ block content
                 if dev.ownerTier !== 'cloud' && dev.ownerTier !== 'global'
                   p= _("This device is owned by your %s.").format(dev.ownerTier)
           div.panel-footer
-            div.row
-              if !dev.isTransient
+            div.row              
```

Comment:
I think at some point you had a change to disable the delete button if the skill is transient (cannot be deleted), but that got lost in the merge.

Reply:
I think you reviewed an old PR version. In the current one, that code line is no longer there.

Reply:
I reviewed out of the "files changed" tab, and I checked on the command-line, the change is still there. Bad merge?

Reply:
I neither see it in "Files Changes" nor in my local repo. Weird...

Reply:
Maybe merge the master branch again? It should clear out what are actual changes in the end.

Reply:
Okay, I just realized, we meant different things 🤦 
I thought you were still seeing the `dev.isTransient` part although it is no longer there. My mistake.

After the redesign of the skill page from #189, my feeling was that the disabled delete button is not really needed anymore. In the original PR version I had added it as a way to ensure that the footer is visually consistent (esp. in terms of height), but that is now achieved without the button.
Given that now a button without a label is used, adding a disabled button would not be helpful IMHO. In the old PR version it was a text button, so we could clearly distinguish between the "Forget" active button and the disabled "Permanent" button with the label.

What do you think?

Reply:
I agree with you on the design, but I think there is an issue with the implementation (not sure if intentional or not). With master, the delete button will be added only if the device is not transient (can be deleted). With the PR as it stands, the delete button will be added unconditionally, which doesn't work.

Reply:
Good point. I think this issue already exists in master today independent of this PR, but I will push a fix now.

Reply:
Good point. I will push a fix.